generated/analysisservices/log.out

```diff
@@ -22,7 +22,11 @@ warning | PreCheck/SchemaMissingType | The schema 'ErrorDetail' with an undefine
 warning | PreCheck/SchemaMissingType | The schema 'ErrorAdditionalInfo' with an undefined type and declared properties is a bit ambiguous. This has been auto-corrected to 'type:object'
     - file:///home/runner/work/bicep-types-az/bicep-types-az/workflow-temp/azure-rest-api-specs/specification/analysisservices/resource-manager/Microsoft.AnalysisServices/stable/2017-08-01/analysisservices.json:1533:5
 warning | PreCheck/CheckDuplicateSchemas | Checking for duplicate schemas, this could take a (long) while.  Run with --verbose for more detail.
-info    | autorest.bicep took 14ms
+warning | Assuming /subscriptions/{subscriptionid}/providers/microsoft.analysisservices/locations/{location}/operationresults/{operationid} is a resource due to the path pattern used. Response definition is missing the x-ms-azure-resource Swagger extension.
```

Comment:
This is a tricky one... `operationResults` is a very common pattern used by RPs to provide means of tracking an async PUT operation. Technically it **is** a GET-only resource, but in a practical sense including it in Bicep is not going to provide any value.

I'm leaning back towards your initial idea of using the `x-ms-azure-resource` extension as a decider - at least that way we can ask RP teams to opt-in by adding it. Again, this may be problematic as this response body technically **is** a resource, but it would be odd for an RP to mark it as such.

Reply:
> I'm leaning back towards your initial idea of using the `x-ms-azure-resource` extension as a decider - at least that way we can ask RP teams to opt-in by adding it if we find we're missing some.

I concur. The [list of probably resource GETs that *don't* use the `x-ms-az-resource` extension](https://gist.github.com/jeskew/332ba3c91cf37216da3cc211d8b31679) is large, but I would prefer to err on the side of adding too few resources and needing to add more later vs adding too many now and needing to remove some in a future release.

Reply:
I've updated and rerun the generator. There are now "only" ~91,000 lines added (the previous iteration added ~104,000). Still a ton of readonly resources added, but these are ones explicitly marked as such by the RP team.